src/estimate.rs

```diff
@@ -165,14 +165,26 @@ pub fn fit<T: Float + From<u32> + From<f64> + Into<f64> + Copy + Add + AddAssign
         }
     }
 
-    let mut fmin = Lbfgsb::new(&mut coef, &f, &g);
-
-    // For debugging
-    // fmin.set_verbosity(101);
-    fmin.set_verbosity(-1);
-    fmin.max_iteration(100);
+    let evaluate = |x: &[f64], gx: &mut [f64]| {
+        let x_vec = x.to_vec();
+        let fx = f(&x_vec);
+        let gx_eval = g(&x_vec);
+        for i in 0..gx_eval.len() {
+            gx[i] = gx_eval[i];
+        }
+        Ok(fx)
+    };
 
-    let result = fmin.minimize();
+    let fmin = lbfgs().with_max_iterations(0);
```

Comment:
Should this be max 100 iterations?
```suggestion
    let fmin = lbfgs().with_max_iterations(100);
```

Reply:
(I guess 100 is kind of random, but it was 100 before)

Reply:
indeed, I have seen no difference  in results ....